read-the-docs-is-hiring-q1-2021.rst

```diff
@@ -0,0 +1,46 @@
+.. post:: Jan 6, 2021
+   :tags: hiring, python, devrel, frontend
+   :author: Eric
+   :location: BND
+
+Read the Docs is hiring for multiple positions
+==============================================
+
+Read the Docs has recently gotten a :doc:`grant to support scientific software</czi-grant-announcement>` in 2021.
+As part of this,
+we are hiring for two new positions related to the grant work:
+
+* A :doc:`frontend developer with design skills </job-frontend>`
+* A :doc:`developer advocate with documentation skills </job-devrel>`
+
+Please look at each of the job postings for much more detail on what to expect with the job.
+
+We are excited to be able to expand our small team and increase the support we're giving to the scientific open source community.
+Throughout 2021 we are going to be vastly improving our documentation and building major new features around embedding documentation content,
+based on the work from this grant.
+
+How hiring will be structured
+-----------------------------
+
+The way we are currently planning to structure this work is to have 50% of the time spent working on grant work,
+and the other 50% working on Read the Docs itself.
+The codebase and community we're building require a good amount of onboarding time,
+so we are excited to be able to use the grant funds to also improve the overall experience of Read the Docs users.
+
+Given that we only have a team of 5 people,
+we need to make sure we are able to support both our existing users and also the new work based on our grant.
+Having some of our existing team doing project management and Python development on the grant means we need additional Read the Docs capacity.
+We currently expect around 50% of one person's time from our existing team to be doing grant work each month,
```

Comment:
This sounds more like an internal note, it's a little confusing as a note to potential applicants.